src/spells.js

```diff
@@ -207,7 +207,7 @@ module.exports = {
     "spellType": "level 8 Transmutation", 
     "duration": "Concentration, up to 8 hours", 
     "castingTime": "10 Minutes", 
-    "shortDescription": "", 
+    "shortDescription": "Needs concentration. Control weather within 5 miles of your location, must be outdoors. Determined by the DM, change the current conditions based on climate and season. After 1d4 x 10 minutes for the condition to take effect, you can change the precipitation, temperature, and wind.", 
```

Comment:
remember, this is going to be read aloud by Alexa. She may not know that when you say "1d4 x 10 minute" you mean 1d4 _times_ 10 minutes.